package.json

```diff
@@ -31,6 +31,7 @@
         "@guardian/automat-client": "^0.2.16",
         "@guardian/consent-management-platform": "^3.0.4",
         "@guardian/discussion-rendering": "^2.2.22",
+        "@guardian/shimport": "^1.0.2",
```

Comment:
Can I just give a big 👏  to whoever came up with this name!

Reply:
Ha, it's actually a fork of https://github.com/Rich-Harris/shimport I made to make it easier to consume. So credit has to go to the original author!